docs/tutorials/federation/set-up-cluster-federation-kubefed.md

```diff
@@ -121,8 +283,19 @@ To use `kubefed join`, you'll need to provide the name of the cluster
 you want to add to the federation, and the `--host-cluster-context`
 for the federation control plane's host cluster.
 
+> Note: The name that you provide to the `join` command is used as the
+joining cluster's identity in federation. This name should adhere to
```

Comment:
We already have a naming rules section below:
Does it make more sense to list all this detail there, and a one liner note here something like:
```Note: The name that you provide to the `join` command is used as the joining cluster's identity in federation.
This name needs to adhere to specific rules, some details of which are listed in naming rules section below:```